frontend/packages/dev-console/src/components/helm/__tests__/HelmReleaseDetailsPage.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import HelmReleaseDetailsPage from '../HelmReleaseDetailsPage';
+
+type HelmReleaseDetailsPageProps = React.ComponentProps<typeof HelmReleaseDetailsPage>;
+let helmReleaseDetailsPageProps: HelmReleaseDetailsPageProps;
+
+describe('HelmReleaseDetailsPage', () => {
+  beforeEach(() => {
+    helmReleaseDetailsPageProps = {
+      secrets: {
+        data: [
+          {
+            metadata: {
+              name: 'secret-name',
+              namespace: 'xyz',
+              creationTimestamp: '2020-01-13T05:42:19Z',
+              labels: {
+                name: 'helm-mysql',
+                owner: 'helm',
+                status: 'deployed',
+              },
+            },
+          },
+        ],
+        loadError: null,
+        loaded: true,
+      },
+      match: {
+        params: {
+          ns: 'xyz',
+        },
+        url: '/helm-releases/ns/xyz/release/helm-mysql',
+        isExact: true,
+        path: '/helm-releases/ns/xyz/release/:name',
+      },
+    };
+  });
+  it('should render the DetailsPage component', () => {
+    const helmReleaseDetailsPage = shallow(
+      <HelmReleaseDetailsPage {...helmReleaseDetailsPageProps} />,
+    );
+    expect(helmReleaseDetailsPage.find(DetailsPage).exists()).toBe(true);
+  });
+  it('should show the ErrorPage404 for an incorrect release name in the url', () => {
```

Comment:
we can have similar tests for `LoadingBox` and `statusBox` as well as conditions are used. WDYT?

Reply:
Added tests for `LoadingBox` and `StatusBox`